### ISSUE TITLE
PrismicNextImage -> height not using correct parameter when not used as explicit parameter

### ISSUE THREAD

**Issue by tijsproost** [prismicio/prismic-next#55]
### Versions

- `@prismicio/next`: 1.0.2
- `next`: ^13.1.0
- node: 16.14.0

### Steps to reproduce

Use the PrismicNextImage object with field={image} where Image is a Prismic image field.

When using an image with layout="responsive" images keep being out of aspect ratio.
In the component inspector I can see that height is equal to width. (My image for instance was 1000px x 200px)

If I use the width & height explicit from image.dimensions.[width/height] then the image scales correctly.

### What is expected?

width & height should be equal to the actual width & height in Prismic when using layout="responsive"

### What is actually happening?

In the component source I can see this code:

```
let resolvedWidth = castedWidth ?? field.dimensions.width;
let resolvedHeight = castedHeight ?? field.dimensions.width;
```

If I'm correct the second line should contain field.dimensions.height instead of the width parameter.

**Comment by jere-co** [comment]
@tijsproost `layout`, including `layout="responsive"`, was removed on Next.js `v13.0.0`. You can see this on the [Version History](https://nextjs.org/docs/api-reference/next/image).

However, this is a bug. 

If you pass either `height` or `width` with any number to the `PrismicNextImage`, it will calculate the other prop using the actual dimensions, but does not work without it.

I'm using a custom component, as the `"use client";` is needed for the Next.js App directory anyway, and I want to have different defaults without repeating them all over the code, so I fixed this there as well for now by spreading the dimensions there: 

```TS
"use client";

import { PrismicNextImage as PNI, type PrismicNextImageProps } from "@prismicio/next";

export const PrismicNextImage = (props: PrismicNextImageProps) => (
  <PNI
    {...props.field?.dimensions}
    {...props}
    quality={props.quality ?? 80}
  />
);
```

**Comment by angeloashmore** [comment]
Hey @tijsproost, thank you for reporting this bug, and sorry it took so long to reply and fix.

This bug is fixed in `v1.0.3` (see PR #61 for more details). You can upgrade to the latest version with the following command:

```sh
npm install --save @prismicio/next@latest
```

If this doesn't fix the issue, please let me know and I'll take another look. Thank you!

---

@jere-co Thanks for sharing your Server Components solution! We are working on official support for RSC in Next.js. We don't have any news or ETA to share yet, but we will post on Twitter (@prismicio) once we have a beta ready to try.
